### PR TITLE
Make font-size: larger/smaller work with the base size stuff

### PR DESCRIPTION
**Pull request by Manishearth**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16730)

### CODE REVIEW

File: components/style/properties/longhand/font.mako.rs [hunk 1]
```diff
@@ -768,6 +768,10 @@ ${helpers.single_keyword_system("font-variant-caps",
                         return Some(em)
                     }
                 }
+            } else if let SpecifiedValue::Larger = *self {
+                return Some(1.2)
+            } else if let SpecifiedValue::Smaller = *self {
+                return Some(1./1.2)
```
Comment (emilio) [inline components/style/properties/longhand/font.mako.rs]: I left two comments on the bug, but I'm actually surprised tidy doesn't complain about the lack of spacing here around the `/`.

**Comment by Manishearth** [comment]
fixing

**Comment by Manishearth** [comment]
git push :(

**Comment by Manishearth** [comment]
@bors-servo retry

LLVM ERROR: Invalid data was encountered while parsing the file

O.o

**Comment by Manishearth** [comment]
@bors-servo retry

android broke

**Pull request merged**